### PR TITLE
Removing playbook tag and using default tag

### PR DESCRIPTION
**Pull request by Shilpi-Das1**

### CODE REVIEW

File: vars/setupCommonEnvironmentVariables.groovy [hunk 1]
```diff
@@ -138,7 +138,7 @@ def call() {
             env.HELPERNODE_REPO = "https://github.com/RedHatOfficial/ocp4-helpernode"
             env.HELPERNODE_TAG = "73312da6192298b3174d54e0a2ae129429561f77"
```
Comment (yussufsh) [inline vars/setupCommonEnvironmentVariables.groovy:139]: We do not need to overwrite default helpernode tag as well.

Comment (Shilpi-Das1) [reply vars/setupCommonEnvironmentVariables.groovy]: After removing tags both PowerVS and PowerVC jobs are working

**Review by yussufsh — APPROVED** [review]
/lgtm

**Review by sudeeshjohn — APPROVED** [review]
/lgtm

**Pull request merged**